### DISCUSSION TITLE
How process MIDI files?

### DISCUSSION THREAD

**Discussion by drscotthawley** [djipco/webmidi#457] [Q&A]
Hi, I'd love to "Kickstart [my] JavaScript MIDI projects!"
My JavaScript MIDI project is: I'm reading in a MIDI file, editing it, and writing it back out.  Uh... I can't find where in `webmidi` you do the file handling.  Where is that?  Thanks.

**Comment by djipco** [comment]
This project is for MIDI communication. MIDI files are a whole other topic. You should like at libraries like [`midi-file`](https://www.npmjs.com/package/midi-file). Cheers!
